src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -94,6 +95,9 @@ export default function (props: SurveyTaskListProps) {
 				if (props.surveys) {
 					parameters.surveyName = props.surveys;
 				}
+				if (props.category) {
+					parameters.surveyCategory = props.category;
```

Comment:
Just want to make a note that we don't want to commit this until we can update the MyDataHelps.js version since this currently gives an error that `surveyCategory` doesn't exist on `SurveyTaskQueryParameters`.